src/server/pkg/hashtree/hashtree_test.go

```diff
@@ -75,8 +63,8 @@ func equals(ltmp, rtmp HashTree) bool {
 
 // requireSame compares 'h' to another hash tree (e.g. to make sure that it
 // hasn't changed)
-func requireSame(t *testing.T, ltmp, rtmp HashTree) {
-	l, r := ltmp.(*hashtree), rtmp.(*hashtree)
+func requireSame(t *testing.T, lTmp, rTmp HashTree) {
+	l, r := lTmp.(*hashtree), rTmp.(*hashtree)
```

Comment:
I think you can use `DeepEqual` directly on interfaces; no need to convert them to concrete types.